src/main/java/pl/put/poznan/buildingInfo/logic/BuildingInfo.java

```diff
@@ -1,21 +1,23 @@
 package pl.put.poznan.buildingInfo.logic;
 
-import java.util.Arrays;
+import java.util.HashMap;
+import java.util.Map;
+
+import pl.put.poznan.buildingInfo.model.Location;
 
 public class BuildingInfo {
 
-    private final String[] buildings;
+    static private HashMap<String, Location> locations = new HashMap<>();
```

Comment:
clever idea with hashmap 👌 i'd be crying, that there is no thing like object in Javascript 😂